.github/workflows/issue-view.yml

```diff
@@ -0,0 +1,384 @@
+                  url: '{{ userdata.url("Azure/powershell","documentation") }}'
+                - value: '{{ userdata.count("Azure/powershell","performance-issue") }}'
+                  url: '{{ userdata.url("Azure/powershell","performance-issue") }}'
+                - value: '{{ userdata.count("Azure/powershell","waiting-for-customer") }}'
+                  url: '{{ userdata.url("Azure/powershell","waiting-for-customer") }}'
+                - value: '{{ userdata.count("Azure/powershell","external-dependency") }}'
+                  url: '{{ userdata.url("Azure/powershell","external-dependency") }}'
+                - value: '{{ userdata.count("Azure/powershell","idle") }}'
+                  url: '{{ userdata.url("Azure/powershell","idle") }}'
+                - value: '{{ userdata.count("Azure/powershell","stale") }}'
+                  url: '{{ userdata.url("Azure/powershell","stale") }}'
+                  
+              - - value: 'sql-action'
+                - value: '{{ userdata.count("Azure/sql-action","") }}'
+                  url: '{{ userdata.url("Azure/sql-action","") }}'
+                - value: '{{ userdata.count("Azure/sql-action","need-to-triage") }}'
+                  url: '{{ userdata.url("Azure/sql-action","need-to-triage") }}'
+                - value: '{{ userdata.count("Azure/sql-action","enhancement") }}'
+                  url: '{{ userdata.url("Azure/sql-action","enhancement") }}'
+                - value: '{{ userdata.count("Azure/sql-action","question") }}'
+                  url: '{{ userdata.url("Azure/sql-action","question") }}'
+                - value: '{{ userdata.count("Azure/sql-action","bug") }}'
+                  url: '{{ userdata.url("Azure/sql-action","bug") }}'
+                - value: '{{ userdata.count("Azure/sql-action","documentation") }}'
+                  url: '{{ userdata.url("Azure/sql-action","documentation") }}'
+                - value: '{{ userdata.count("Azure/sql-action","performance-issue") }}'
+                  url: '{{ userdata.url("Azure/sql-action","performance-issue") }}'
+                - value: '{{ userdata.count("Azure/sql-action","waiting-for-customer") }}'
+                  url: '{{ userdata.url("Azure/sql-action","waiting-for-customer") }}'
+                - value: '{{ userdata.count("Azure/sql-action","external-dependency") }}'
+                  url: '{{ userdata.url("Azure/sql-action","external-dependency") }}'
+                - value: '{{ userdata.count("Azure/sql-action","idle") }}'
+                  url: '{{ userdata.url("Azure/sql-action","idle") }}'
+                - value: '{{ userdata.count("Azure/sql-action","stale") }}'
+                  url: '{{ userdata.url("Azure/sql-action","stale") }}'
+        token: ${{ github.token }}
+
+    - name: Commit files
+      id: commit
+      continue-on-error: true
+      run: |
+        git config --local user.email "kanika1894@github.com"
+        git config --local user.name "kanika1894"
```

Comment:
git config can be updated to :
git config --local user.email "action@github.com"
git config --local user.name "GitHub Action"

Reply:
updated Thanks :)

Reply:
here, any user email name can be given?

Reply:
This we used for testing in kanika's repository, already changed config here.

Reply:
@kaverma This is just for setting the git configuration so that we can run git commands.
Took reference from other actions and used the same email name .
Reference : https://github.com/Azure/arm-deploy/blob/main/.github/workflows/build-release.yaml